ee/tabby-webserver/email_templates/invitation.html

```diff
@@ -1,4 +1,5 @@
 You've been invited to join a Tabby workspace!
+---
 Welcome to Tabby! You have been invited to join a Tabby Server, where you can tap into AI-driven code completions and chat assistants.
 
 Go to {external_url}/auth/signup?invitationCode={code} to join!
```

Comment:
```suggestion
Go to {{EXTERNAL_URL}}/auth/signup?invitationCode={{INVITATION_CODE}} to join!
```

nit: capitalize the variables, and use `{{`, `}}` to be compatible with mustache template engine, just in case we'd like to adopt one in future